### PR TITLE
Rating

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Screenshot (90)
Screenshot (91)
Screenshot (92)

This is the Rating component of my CL. Please review it!

### CODE REVIEW

File: Components/Rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    justify-content: center;
+    align-items: center;
+    border-radius: var(--container-border-radius);
+    margin: var(--container-margin);
+}
+
+.star-section {
+    display: flex;
+    background-color: var(--bg-color);
+    gap: 0.25rem;
```
Comment (iRohitGaur) [inline Components/Rating/rating.css]: suggestion: this should be in px. value too small to be a relative unit

File: Components/Rating/rating.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    display: flex;
+    flex-direction: column;
+    gap: var(--container-gap);
+    background-color: var(--box-color);
+    width: var(--container-width);
+    height: 15%;
+    justify-content: center;
+    align-items: center;
+    border-radius: var(--container-border-radius);
+    margin: var(--container-margin);
```
Comment (dev-ashishsethi) [inline Components/Rating/rating.css:11]: Good that you are using variables for sizes as well good work on consistency

File: Components/Rating/rating.html [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+                <li class="list-menu">
+                    <a href="../../Get Started/color.html">Colors</a>
+                </li>
+            </ul>
+            <h3 class="list-menu list-menu-no-hover">Components</h3>
+            <div class="line"></div>
+            <ul>
+                <li class="list-menu">
+                    <a href="../Avatar/avatar.html">Avatar</a>
+                </li>
```
Comment (dev-ashishsethi) [inline Components/Rating/rating.html:59]: really liked your class naming pattern its easy to understand

File: Components/Rating/rating.html [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+                    <div class="star-section">
+                        <button><i class="fas fa-star fa-2x filled"></i></button>
+                        <button><i class="fas fa-star fa-2x filled"></i></button>
+                        <button><i class="fas fa-star fa-2x unfilled"></i></button>
+                        <button><i class="fas fa-star fa-2x unfilled"></i></button>
+                        <button><i class="fas fa-star fa-2x unfilled"></i></button>
+                    </div>
+            </script>
+            </div>
+            <div class="line w-800"></div>
```
Comment (dev-ashishsethi) [inline Components/Rating/rating.html:131]: could have used hr tag instead of div

**Review by dev-ashishsethi — APPROVED** [review]
good naming conventions followed but do check comments

File: Components/Rating/rating.css [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+.star-section {
+    display: flex;
+    background-color: var(--bg-color);
+    gap: 0.25rem;
+    padding: 0.75rem;
+    border-radius: 0.25rem;
+}
+
+.filled {
+    color: #FDCC0D;
```
Comment (iRohitGaur) [inline Components/Rating/rating.css:23]: suggestion: use variables everywhere for consistency

**Review by iRohitGaur — APPROVED** [review]
Nice work!

Clean code. Easy to understand.

Good to merge 👍🏻

**Review by zoyatanweer — APPROVED** [review]
Clean code, nice use of variables and I liked the naming of the components. Good to merge with suggested changes!

**Review by sumitk-singh — APPROVED** [review]

File: Components/Rating/rating.css (hunk 1 above)
Comment (Megha-Joshi) [reply Components/Rating/rating.css]: Done

File: Components/Rating/rating.html (hunk 4 above)
Comment (Megha-Joshi) [reply Components/Rating/rating.html:131]: okay will do it.

**Pull request merged**